arch/arm/src/stm32h7/hardware/stm32h7x3xx_pinmap.h

```diff
@@ -728,10 +728,10 @@
 
 /* USB on-the-go full-speed (OTG_FS) */
 
-#define GPIO_OTGFS_DM_0           (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN11)
-#define GPIO_OTGFS_DP_0           (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN12)
-#define GPIO_OTGFS_ID_0           (GPIO_ALT|GPIO_AF10|GPIO_OPENDRAIN|GPIO_PULLUP|GPIO_PORTA|GPIO_PIN10)
-#define GPIO_OTGFS_SOF_0          (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN8)
+#define GPIO_OTGFS_DM           (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN11)
+#define GPIO_OTGFS_DP           (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN12)
+#define GPIO_OTGFS_ID           (GPIO_ALT|GPIO_AF10|GPIO_OPENDRAIN|GPIO_PULLUP|GPIO_PORTA|GPIO_PIN10)
+#define GPIO_OTGFS_SOF          (GPIO_ALT|GPIO_AF10|GPIO_PUSHPULL|GPIO_FLOAT|GPIO_PORTA|GPIO_PIN8)
```

Comment:
That is also incorrect, the new pinmap format requires the _0 at ending and you need to define the GPIO_OTGFS_DM at your board.h like this:
```
#define GPIO_OTGFS_DM  (GPIO_OTGFS_DM_0|GPIO_SPEED_100MHz) /* PA11 */
```

Please use the linum-stm32h753bi as reference